tpch-scripts/gettpchtablecounts.sql

```diff
@@ -0,0 +1,15 @@
+use ${DB};
+
+SELECT COUNT(*) FROM part
+ 	UNION ALL
```

Comment:
Why combine them under union all, instead of individual table counts ? In the current form, we will not detect if some table has more rows and other less - canceling out each other. Also, would not know which table is a problem in case the counts don't match.